### ISSUE TITLE
paging: stops working at page 19, so page 20 doesnt show anything!

### ISSUE THREAD

**Issue by haschimoto** [zer0net/zerovoat#183]
I know there are older posts on page 20 and older, since there are many missing posts like "Is es denn die possibility"

**Comment by dnelband** [comment]
on my local client there are 26 pages, i see that on 0net many other topics are missing, im not sure why but i think because its a proxy. on your local client are there only 19 pages or more?

**Comment by haschimoto** [comment]
Yeah, it seems fine, but also topics are simply missing, also viceversa, so
there are new topics on proxy that dont load on local... not sure why.
